docs-chef-io/content/inspec/plugins.md

```diff
@@ -78,7 +77,7 @@ inspec plugin install --source https://my.private.server inspec-private-plugin
 ### Chef InSpec Plugins
 
 For details on how to author a Chef InSpec Plugin, see the
-[developer documentation](https://github.com/inspec/inspec/blob/master/docs/dev/plugins.md)
+[developer documentation](https://github.com/inspec/inspec/blob/master/dev-docs/plugins.md)
```

Comment:
This link was broken, it appears to be referring to [this document](https://github.com/inspec/inspec/blob/master/dev-docs/plugins.md) so I updated the link.